cmd/opm/registry/serve.go

```diff
@@ -90,6 +90,10 @@ func serveFunc(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if _, err := db.ExecContext(context.TODO(), `PRAGMA soft_heap_limit=1`); err != nil {
```

Comment:
In case anyone is unfamiliar with this setting, https://www.sqlite.org/c3ref/hard_heap_limit64.html (includes info on `soft_heap_limit`)